server/src/main/java/p2/dao/impl/UserDAO.java

```diff
@@ -1,8 +1,26 @@
 package p2.dao.impl;
 
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+
 import p2.dao.IUserDAO;
 import p2.model.User;
+import p2.util.HibernateUtil;
+
+public class UserDAO extends GenericDAO<User> implements IUserDAO {
+
+	public User findByEmailANDPassword(String email, String password) {
```

Comment:
can we call this method findByEmailAndPassword?

Reply:
done